jwst/tests_nightly/general/nirspec/test_nrs1_fs_wcs.py

```diff
@@ -21,35 +19,25 @@ def test_nirspec_nrs1_wcs(_bigdata):
     Regression test of creating a WCS object and doing pixel to sky transformation.
 
     """
-    output_file_base, output_file = add_suffix('nrs1_fs_wcs_output.fits', 'assignwcsstep')
-
-    try:
-        os.remove(output_file)
-    except:
-        pass
-
     input_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-fs', 'jw00023001001_01101_00001_NRS1_ramp_fit.fits')
-    ref_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-fs', 'jw00023001001_01101_00001_NRS1_assign_wcs.fits')
+    ref_file = os.path.join(_bigdata, 'nirspec', 'test_wcs', 'nrs1-fs', 'jw00023001001_01101_00001_NRS1_ramp_fit_assign_wcs.fits')
 
-    AssignWcsStep.call(input_file,
-                       output_file=output_file_base, name='assignwcsstep'
+    result = AssignWcsStep.call(input_file,
```

Comment:
nitpick: did an IDE allow this indentation? Though allowed, not a standard. Either:
```
    AssignWcsStep.call(
        input_file,
        output_file=output_file_base,
        name='assignwcsstep'
    )
```
which is my preference, or
```
    AssignWcsStep.call(input_file,
                       output_file=output_file_base,
                       name='assignwcsstep')
```

Reply:
Yes, that is ugly.  Will fix.

Reply:
@stscieisenhamer That's not what PEP 8 says.  There are examples showing multiple arguments on each line when aligning after the '('.